### DISCUSSION TITLE
Equipo 18 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#28] [Proyecto Final]
Hola @Syndelcallisaya 

Aquí está el hilo para coordinar tu proyecto, charlar con el Equipo de Educación, recibir retroalimentación sobre cada etapa.

Aquí está su carpeta grupal en nuestro repositorio compartido: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/tree/main/equipo-18
La carpeta tiene formularios de entrega para cada etapa.

Con la emoción de seguir el desarrollo de su proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Syndelcallisaya 

Espero estés muy bien. ¿Cómo andas con tu proyecto final?

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar su presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si les conviene la asignación de su equipo a una de las dos sesiones de clausura del curso, o si quieren cambiar su día asignado al otro día. Horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing))

**_No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo._**

(Una recomendación: La etapa 5 no necesita que la etapa 3 ni 4 sea completada. Entonces, puedes pasar 30 minutos (es corta) completando la etapa 5, entregarla, y luego avanzar con las etapas 3 y 4.)

Cualquier pregunta sobre el proyecto final, déjanos saber. @marijoct y yo estamos aquí para apoyarte. Quedamos atentas a tus actualizaciones para poder revisar las etapas / apoyarte en lo que podamos.

Mucha suerte, mucho éxito, hasta la clausura del curso,
Kasturi

**Comment by Syndelcallisaya** [comment]
Buenos días querida Kasturi ☺.
Voy a completar el proyecto y acabar lo que me falta de la plataforma.
Te escribiré para informarte como me va.

Saludos, Syndel.

**Reply by KasturiSShah** [reply]
Hola @Syndelcallisaya buenos días, espero estés muy bien. Un gusto en saludarte. Listo, me parece genial. Avísame cualquier cosa. 
Saludos, nos vemos durante la clausura,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Syndelcallisaya  

Espero estés muy bien. Te escribo para preguntarte ¿si quieres presentar el sábado durante el cierre del curso? Te agendaré a presentar el sábado con gusto, si quieres.

Anteriormente te asignamos a presentar ayer, miércoles. Ya que no te vi en la sesión, quisiera ofrecer otra oportunidad 😄 

Quedo atenta, lindo día,
Kasturi

**Comment by Syndelcallisaya** [comment]
Buenos días Kasturi.
Lo siento mucho, no ví que me tocaba el miércoles justo queria anotarme
para el sábado por favor.

**Reply by KasturiSShah** [reply]
Hola @Syndelcallisaya 
No te preocupes, te he agendado a presentar tu proyecto final mañana, sábado 29 de abril. Horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing).
¡Hasta mañana!
Kasturi
